docroot/WEB-INF/src/edu/osu/cws/evals/tests/BackendMgrTests.java

```diff
@@ -326,6 +326,43 @@ public void shouldSendProfFacultyEmailsRightAwayOnceReviewPeriodStarts() throws
         assert !BackendMgr.timeToSendFirstStatusEmail(appraisal, new DateTime());
     }
 
+    public void shouldUpdateSalaryWhenAppraisalIsDue() throws Exception {
+        Appraisal appraisal = new Appraisal();
+        appraisal.setJob(new Job());
+        appraisal.getJob().setAppointmentType(AppointmentType.CLASSIFIED_IT);
+        DateTime endDate = new DateTime().plusDays(61).withTimeAtStartOfDay();
+        appraisal.setEndDate(endDate.toDate());
+        appraisal.getSalaries().add(new Salary());
+
+        // get instance and call updateAppraisal
+        Session session = HibernateUtil.getCurrentSession();
+        Transaction tx = session.beginTransaction();
+        BackendMgr mgr = getMgrInstance();
+        session = HibernateUtil.getCurrentSession();
+        tx = session.beginTransaction();
+
+        assert mgr.shouldUpdateSalaryInfo(appraisal) : "Should refresh since appraisal is due in one day";
```

Comment:
Not quite. The configurations are confusing in this application. The appraisal is due 60 days before the appraisal.endDate. That's why by setting the end date to be 61 days from today, makes it due in one day, 60 days from today means due today and 59 means it was due yesterday.